tests/cl-go-client/main.go

```diff
@@ -104,8 +105,16 @@ func main() {
 			)
 
 			log.Println("creating position: pool id", expectedPoolId, "accountName", accountName, "lowerTick", lowerTick, "upperTick", upperTick, "token0Desired", tokenDesired0, "tokenDesired1", tokenDesired1, "defaultMinAmount", defaultMinAmount)
-			amt0, amt1, liquidity := createPosition(igniteClient, expectedPoolId, accountName, lowerTick, upperTick, tokenDesired0, tokenDesired1, defaultMinAmount, defaultMinAmount)
-			log.Println("created position: amt0", amt0, "amt1", amt1, "liquidity", liquidity)
+
+			maxRetries := 100
+			for j := 0; j < maxRetries; j++ {
+				amt0, amt1, liquidity := createPosition(igniteClient, expectedPoolId, accountName, lowerTick, upperTick, tokenDesired0, tokenDesired1, defaultMinAmount, defaultMinAmount)
+				if err == nil {
+					log.Println("created position: amt0", amt0, "amt1", amt1, "liquidity", liquidity)
+					break
+				}
+				time.Sleep(8 * time.Second)
```

Comment:
Why 8 seconds in particular? Would it potentially run faster if the sleep was for fewer seconds?

Reply:
No specific reason, it's close to the estimated block time.
Most of the reason why the program would fail would be because we would be having too many transactions in a single block and get sequence mismatch. Thus the original intent of sleeping 8 second was to have the process sleep for 8 second and retry in the next block